### PR TITLE
Removed C style arrays from the BlockEntity folder

### PR DESCRIPTION
**Pull request by tigerw**

### CODE REVIEW

**Review by 12xx12 — CHANGES_REQUESTED** [review]
Code looks good.

I know it's a bit nitpicky, but I think we should update the variable names to the new NamingSceme when touching the code of older functions.
If you think otherwise dismiss this review

**Comment by tigerw** [comment]
yes I agree. Waiting on #4979 to see if I should close this or not.

File: src/BlockEntities/BrewingstandEntity.cpp [hunk 1]
```diff
@@ -130,12 +124,12 @@ bool cBrewingstandEntity::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 		const cBrewingRecipes::cRecipe * Recipe = nullptr;
 		for (int i = 0; i < 3; i++)
```
Comment (12xx12) [inline src/BlockEntities/BrewingstandEntity.cpp:125]: Why not use a `std::size_t` as the iteration variable

File: src/BlockEntities/BrewingstandEntity.cpp [hunk 2]
```diff
@@ -254,14 +248,14 @@ void cBrewingstandEntity::OnSlotChanged(cItemGrid * a_ItemGrid, int a_SlotNum)
 	{
 		if (GetSlot(i).IsEmpty())
 		{
-			m_CurrentBrewingRecipes[i] = nullptr;
-			m_Results[i].Clear();
+			m_CurrentBrewingRecipes[static_cast<size_t>(i)] = nullptr;
+			m_Results[static_cast<size_t>(i)].Clear();
```
Comment (12xx12) [inline src/BlockEntities/BrewingstandEntity.cpp:252]: Same here

File: src/BlockEntities/BrewingstandEntity.cpp [hunk 3]
```diff
@@ -348,8 +342,8 @@ void cBrewingstandEntity::LoadRecipes(void)
 		Recipe = BR->GetRecipeFrom(GetSlot(i), GetSlot(bsIngredient));
 		if (Recipe != nullptr)
 		{
-			m_CurrentBrewingRecipes[i] = Recipe;
-			m_Results[i] = Recipe->Output.CopyOne();
+			m_CurrentBrewingRecipes[static_cast<size_t>(i)] = Recipe;
+			m_Results[static_cast<size_t>(i)] = Recipe->Output.CopyOne();
```
Comment (12xx12) [inline src/BlockEntities/BrewingstandEntity.cpp:346]: And here

**Review by 12xx12 — CHANGES_REQUESTED** [review]
Here I am after a long time going through pull requests.

There is another thing I saw

**Comment by tigerw** [comment]
what a dereliction of duty, i've left this for two years! thanks for the comments :D

**Comment by 12xx12** [comment]
Fixed it myself and updated it to master

**Comment by 12xx12** [comment]
I think I crashed the CI. Again.... xD

**Comment by bearbin** [comment]
Apologies for the broken CI, it's not you it's just OVH's broken openStack implementation. I really ought to sort it out, but never got the time. Maybe soon :)  I have poked it so the queued up builds should be going now.

**Review by bearbin — APPROVED** [review]
Fine, GTM when the build passes.

**Pull request merged**